### DISCUSSION TITLE
Add sections to the Ingredients

### DISCUSSION THREAD

**Discussion by werner-bros** [mealie-recipes/mealie#2595] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

For each recipe it is only possible to have one Ingredients list.

![ingredients](https://github.com/mealie-recipes/mealie/assets/82870917/bfd96428-8f3a-4841-9e9b-a63bd8f8e039)

### Please provide a concise description of the feature that would resolve your issue.

I would love to see sections / categories for the Ingredients List. What I mean by that is, if you have a recipe, e.g. Pizza, with different parts, such as the dough, the topping, the dip, to be able to have two or more Ingredients Lists, such as with the steps to cooking, so you could sort the ingredients by:
For the dough:
400g flour
Yeast
Water

For the topping:
Tomato sauce
Cheese

For the Dip:
Creme Fraiche
Minced Garlic
etc.

### Please consider and list out some caveats or tradeoffs made in your design decision

I don't know, how easy or difficult it would be, to implement a feature like this. I'd also be happy to contribute or donate, but I only have little experience with coding and little money (I'm a student). 

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by JvdMaat** [comment]
I think this is what you're looking for:
![image](https://github.com/mealie-recipes/mealie/assets/90044149/73d3fb4f-18af-49c1-8166-8b4c1d007ee6)

**Comment by JvdMaat** [comment]
(This is already in Mealie, and the instructions also have the Toggle Section option)

**Comment by werner-bros** [comment]
Thank, I'm sorry I missed that
